app/controllers/api/functions.php

```diff
@@ -461,6 +469,13 @@
 
         $enabled ??= $function->getAttribute('enabled', true);
 
+        $needToDeploy = false;
```

Comment:
Somewhere here we should try to get Installation document. then do `if($installation->isEmpty())`, and throw error if it's empty.